bin/sngl/pycbc_export_censored_trigs

```diff
@@ -102,8 +105,39 @@ else:
     trigs_censored = ForegroundTriggers(opts.coinc_file, opts.bank_file,
                           sngl_files=opts.sngl_files, group=group)
 
-# find triggers above limit
-mask = numpy.where( trigs_censored.get_coincfile_array("stat") > opts.min_stat )[0]
+# apply clustering to censored_triggers if a cluster-window has been parsed in the command line else skip clustering. Then pick triggers above min-stat if one is parsed in the command line
```

Comment:
Also one sort of pedantic, nit-picky thing. We try to use the PEP-8 style guide (https://www.python.org/dev/peps/pep-0008/). So lines should try to be limited to 72 characters.

You can set up vim to highlight characters past 72 characters as a sort of alert.

Can you make this comment span two lines? Its fairly long.